go/cmd/dolt/commands/merge.go

```diff
@@ -480,17 +480,17 @@ func calculateMergeStats(queryist cli.Queryist, sqlCtx *sql.Context, mergeStats
 		if strings.HasPrefix(summary.TableName.Name, diff.DBPrefix) {
 			continue
 		}
-		if summary.DiffType == "added" {
+		if summary.DiffType == diff.DiffTypeAdded {
```

Comment:
FYI, used the new diff type constants added here so we aren't used string literals here